tools/slang-test/slang-test-main.cpp

```diff
@@ -1438,12 +1412,16 @@ TestResult runCompile(TestContext* context, TestInput& input)
     CommandLine cmdLine;
     _initSlangCompiler(context, cmdLine);
 
+    StringEscapeHandler* escapeHandler = StringEscapeUtil::getHandler(StringEscapeUtil::Style::Space);
+
     for (auto arg : input.testOptions->args)
     {
         // If there is a quote in the string, assume it is 'escaped'. 
-        if (arg.indexOf('"') >= 0)
+        if (arg.indexOf(escapeHandler->getQuoteChar()) >= 0)
```

Comment:
Why is this logic not part of StringEscapeUtil? I feel like there are a lot of design complexity involved around EscapeHandler, but most use sides really just want to deal with a simple set of helper functions.